modules/mobile/spec/support/vcr_cassettes/appointments/VAOS_v2/get_appointments_no_requests_200.yml

```diff
@@ -2,7 +2,7 @@
 http_interactions:
 - request:
     method: get
-    uri: https://veteran.apps.va.gov/vaos/v1/patients/1012846043V576341/appointments?end=2023-01-01T00:00:00Z&pageSize=0&start=1991-01-01T00:00:00Z&statuses=cancelled,booked,fulfilled
+    uri: https://veteran.apps.va.gov/vaos/v1/patients/1012846043V576341/appointments?end=2023-01-01T00:00:00Z&pageSize=0&start=1991-01-01T00:00:00Z&statuses=proposed,cancelled,booked,fulfilled
```

Comment:
I can give that a try.

Reply:
This was a good point you raised. I ended up changing the cassette back and using a different cassette that included proposed instead as you'll see in the specs. Do you think there's any reason I should add a param to the new interface class to allow us to exclude certain appointment types (like proposed)? For V0, we always fetch appointment requests if the appointment requests feature flag is on. So we could add that feature flag to the v2 proxy because (presumably) we'll be turning on V2 before we turn on appointment requests.

Reply:
I like the idea of keeping it as it is where `include_pending` is always true. This makes caching a lot easier. It feels right to have the controller do the filtering out.